### DISCUSSION TITLE
Starting Tips

### DISCUSSION THREAD

**Discussion by rzhao621** [mmistakes/jekyll-theme-basically-basic#159] [Q&A]
Do you have any pointers for those who are new to Jekyll and GitHub? I have read through lots of documentation and watched video tutorials but am having trouble getting a proper website running using this theme. I tried creating website using "jekyll new" through Git but have been more successful getting a website running by just forking this repos.

**Comment by lizsugar** [comment]
What exactly trouble are you having?  I was able to apply the theme to my existing jekyll site, using rubygems with minimal effort.
